adoc/deployment-sysreqs.adoc

```diff
@@ -47,25 +47,29 @@ In order to deploy and control a {productname} cluster you will need at least on
 machine capable of running `skuba`. This typically is a regular desktop workstation or laptop
 running SLE 15 SP1 or later.
 
-[NOTE]
-====
 The `skuba` CLI package is available from the {productname} module.
 You will need a valid {sle} and {productname} subscription to install this tool on the workstation.
-====
 
-.Storage Sizing
+
+.Time Synchronization
 [IMPORTANT]
 ====
+It is vital that the management workstation runs an NTP server and that time synchronization is configured to the same NTP servers,
```

Comment:
I don't understand the line break here in the middle of the sentence. Was that by accident?

Reply:
And the wording here is wrong. The workstation doesn't have to provide an NTP server to the cluster nodes. It must be configured so that the local NTP client on the workstation is getting time synchronization from the same servers that will be used for the cluster nodes.

We recommend the customer to provide an NTP server in their network but we don't want to explain how. All we need to convey is that all the NTP clients across the machines must use the same servers in order to get the same timing.